workers/ohsome_quality_analyst/indicators/ghs_pop_comparison_roads/indicator.py

```diff
@@ -19,7 +20,10 @@ class GhsPopComparisonRoads(BaseIndicator):
     """Set number of features and population into perspective."""
 
     def __init__(
-        self, layer: Layer, feature: Feature, thresholds: tuple = None
+        self,
+        layer: Layer,
+        feature: Feature,
+        thresholds: Optional[Tuple[float, float, float, float]] = None,
```

Comment:
For this indicator its actualy a tuple of dicts (see in the layer-thresholds in the metadata:
`[big-data/ohsome](default: [{ a: 1000 }, null, { a: 500 }, null])`

Here we also need to check the threshold function used by this indicator to use the threshold function parameter

Reply:
done